app/services/idv/steps/in_person/address_step.rb

```diff
@@ -9,11 +9,19 @@ def self.analytics_visited_event
         end
 
         def self.analytics_submitted_event
-          :idv_in_person_proofing_address_submitted
+          if IdentityConfig.store.in_person_capture_secondary_id_enabled
+            :idv_in_person_proofing_residential_address_submitted
+          else
+            :idv_in_person_proofing_address_submitted
+          end
         end
 
         def call
           Idv::InPerson::AddressForm::ATTRIBUTES.each do |attr|
+            if attr == :same_address_as_id && IdentityConfig.store.
```

Comment:
checking my understanding - here we're saying if the current residential address is the same as the one on the user's id don't get attributes from `flow_params`

Reply:
No, we're choosing whether to assign the `same_address_as_id` field to the session here based on the feature flag. `#call` here is accepting data from the address form submission.